lymph/unilateral.py

```diff
@@ -260,27 +276,27 @@ def base_probs(self):
         values, the transition matrix - if it was precomputed - is deleted
         so it can be recomputed with the new parameters.
         """
-        return np.array([edge.base_t for edge in self.tumor_edges], dtype=float)
+        return np.array([edge.spread_prob for edge in self.tumor_edges], dtype=float)
 
-    @base_probs.setter
-    def base_probs(self, new_base_probs):
+    @tumor_spread_probs.setter
+    def tumor_spread_probs(self, new_tumor_probs):
         """Set the spread probabilities for the connections from the tumor to
         the LNLs.
         """
         for i, edge in enumerate(self.tumor_edges):
-            edge.base_t = new_base_probs[i]
+            edge.spread_prob = new_tumor_probs[i]

```

Comment:
Maybe iterating via `zip()` here is safer. Now, if someone provides too few `new_tumor_probs`, an error is raised. `zip()` only iterates up the the shortest of its arguments. Of course, one could argue that then we don't notice if something is left unassigned, but if it's mentioned in the docstring, I think it's fine.

Reply:
This one should disappear anyway with the new spread_prob setting. 
It was only a bypassing solution to check whether the whole thing runs properly :)

Reply:
I agree that all parameters should be set via `assign_parameters`, but I also see a value in being able to use _both_ an array/list and a dictionary for updating the values.